app/controllers/application_controller.rb

```diff
@@ -49,8 +49,8 @@ def redirect_to_signin
     redirect_to sign_in_path, alert: t("please_sign_in")
   end
 
-  def mfa_required?
-    current_user&.mfa_required?
+  def mfa_non_compliant?
```

Comment:
I'm open to suggestions for naming this concept. Alternatives I thought of were:
- invalid_mfa_config
- invalid_mfa_configuration
- invalid_mfa_setting
- non_compliant_mfa
- mfa_non_compliant